package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@tizentv/tools",
+    "version": "0.0.1",
+    "description": "Common APIs for tizen tools download and usage.",
+    "main": "./lib/tools.js",
+    "author": "Samsung Electronics Co., Ltd.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Samsung/tools.git"
+    },
+    "dependencies": {
+        "npm-conf": "^1.1.3",
+        "got": "^11.5.2",
+        "hpagent": "^0.1.1",
+        "decompress": "4.2.1"
```

Comment:
There is no xml2js.
And please keep in mind other modules also should be added.

Reply:
done